.github/workflows/stale_bot.yml

```
@@ -0,0 +1,49 @@
+# SPDX-License-Identifier: Apache-2.0
+
+name: 'Close stale issues and PRs'
+
+on:
+  schedule:
+    - cron: '30 1 * * *'
+
+env:
+  LC_ALL: en_US.UTF-8
+
+defaults:
+  run:
+    shell: bash
+
+permissions:
+  contents: read
+
+jobs:
+  stale:
+    permissions:
+      issues: write
+      pull-requests: write
+    runs-on: ubuntu-latest
+    steps:
+      - name: "Harden Runner"
+        uses: step-security/harden-runner@5c7944e73c4c2a096b17a9cb74d65b6c2bbafbde # v2.9.1
+        with:
+          egress-policy: audit # TODO: change to 'egress-policy: block' after couple of runs
```

```suggestion
          disable-sudo: true
          egress-policy: block
          allowed-endpoints: >
            api.github.com:443
```

For these stale workflows, we can configure the egress policy as above.

I should get around to making PRs to tighten this up for all the workflows in all the repos.

---

Will make this change!